x/bank/keeper/view.go

```diff
@@ -194,7 +194,23 @@ func (k BaseViewKeeper) LockedCoins(ctx context.Context, addr sdk.AccAddress) sd
 // by address. If the account has no spendable coins, an empty Coins slice is
 // returned.
 func (k BaseViewKeeper) SpendableCoins(ctx context.Context, addr sdk.AccAddress) sdk.Coins {
-	spendable, _ := k.spendableCoins(ctx, addr)
+	total := k.GetAllBalances(ctx, addr)
```

Comment:
Can we add a comment to the function that describes exactly how this works?

Reply:
i.e. 

What we do when we have:
- no locked coins
- more locked than in balances
- some locked, some unlocked

Reply:
I updated with specific flow